chanreg.md

```diff
@@ -1,228 +1,196 @@
+  presence on Libera.Chat. Regardless of the setting you choose, group
+  contacts will receive notice of all channels registered under their
+  namespace and can audit their channel list at any time with
+  `/msg ChanServ LISTGROUPCHANS`.
+- Group contacts will be able request cloaks for their group's members which
+  highlight that member's involvement or association with their group. For
+  example, Libera.Chat staff each have cloaks beginning with `libera/staff/`
+  and official bots managed by Libera.Chat have cloaks beginning with
+  `libera/bot/` to represent their respective associations with Libera.Chat.
+  Formal community registration cloaks will begin with an `about/` prefix to
+  indicate that the community is not official. Group contacts can audit the
+  list of users with cloaks under their project at any time with
+  `/msg NickServ LISTGROUPCLOAKS`.
+- We offer to send messages about important releases or events of formally
+  registered project as a "wallop" message to our users.
+
+### Group contacts
+
+Group contacts are the people ultimately responsible for your project here on
+Libera.Chat. They are empowered to claim channels within your namespace,
+assign cloaks to other users, and act as liaisons between Libera.Chat staff
+and your project. We **strongly recommend** having between 2-4 group contacts
+and that at least one group contact is publicly listed to act as a point of
+contact for any users interested in your project.
+
+Group contacts must have registered accounts with NickServ on Libera.Chat.
+Usually, you will want to establish yourself as one of the group contacts if
+you are formally registering a group with us. The list of group contacts for
+your project or community can be modified later by sending an email to
+<projects@libera.chat> as the need arises.
+
+Group contacts can be either publicly listed or hidden, and this choice can be
+made individually per group contact. Publicly listed contacts will be shown in
+`/msg ChanServ INFO` output for all channels within your group's channel
+namespaces. Hidden group contacts are known only to you and Libera.Chat staff.
+
+### Registering a group
+
+The [Projects & Community team](/about#projects-and-community-team) will act
+as your liaisons during the formal group registration process and throughout
+your tenure on the Libera.Chat network. If you wish to discuss the
+registration process, find out if Libera.Chat could be a good fit for your
+group or register your group, please feel free to drop any of the team members
+a line on IRC. If none are online, feel free to reach out to staff in general
+and they will forward your request to the team, which will get back to you as
+soon as possible. Once you have discussed your project or community with a
+staff member, you’ll be asked to email <projects@libera.chat> with the details
+of your request.
+
+We ask that you do not email us without having first discussed your
```

Comment:
I'd love to convey to people that if they email in without talking to anyone, that no one will be looking for or particularly worrying about their submission.